### PR TITLE
skip_scratch option for add_lines

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

**Comment by twaugh** [comment]
See #68 for the fix for the test-case failure on Python 2.6.

File: dockerfile_parse/parser.py [hunk 1]
```diff
@@ -597,11 +597,13 @@ def add_lines(self, *lines, **kwargs):
         :param all_stages: bool for whether to add in all stages for a multistage build
                            or (by default) only the last.
         :param at_start: adds at the beginning (after FROM) of the stage(s) instead of the end.
+        :param skip_scratch: skip for FROM scratch
```
Comment (twaugh) [inline dockerfile_parse/parser.py]: Would be nice to make this slightly more explicit, like: `skip stages which use "FROM scratch"`

Comment (rcerven) [reply dockerfile_parse/parser.py]: ok

**Comment by rcerven** [comment]
we don't want to add lines with yum repos to FROM scratch sections

**Comment by MartinBasti** [comment]
> we don't want to add lines with yum repos to FROM scratch sections

IIUC correctly, you are updating underlying library to not load `FROM scratch` sections, to prevent accidental updates of yum repos in wrong section in atomic-reactor(?), instead of handling this logic inside atomic-reactor(?) ?

**Comment by rcerven** [comment]
rn updated:
* skip_scratch option for add_lines, to skip adding lines to FROM scratch sections

**Pull request merged**